src/pages/Commits.tsx

```diff
@@ -200,13 +201,15 @@ export const Commits = () => {
             nextPageOrderNumber={nextPageOrderNumber}
           />
         </PaginationWrapper>
-        <CommitsWrapper
-          versions={versions}
-          error={error}
-          isLoading={loading || !projectIdentifier}
-          hasTaskFilter={hasTasks}
-          hasFilters={hasFilters}
-        />
+        <span ref={ref}>
```

Comment:
```suggestion
        <div ref={ref}>
```

I think a div is the more "semantically correct" wrapper since `CommitsWrapper` represents several block components.